docs/introduction/Car-registry-tutorial.rst

```diff
@@ -413,14 +408,13 @@ Few notes: setter’s input parameter could have a different type than set class
         }
     }
 
-
 * Define Car creation transaction itself
   ::
    private ApiResponse createCar(SidechainNodeView view, CreateCarBoxRequest ent)
 
 As a first parameter we pass reference to SidechainNodeView, second reference is previously defined class on step 1 for representation of JSON request. 
 
-* Define request for Car sell order transaction CreateCarSellOrderRequest  similar as it was done for Car creation transaction request
+* Define request for Car sell order transaction CreateCarSellOrderRequest as it was done similarly for Car creation transaction request
```

Comment:
@zennermac 

* Define the request for the CarSellOrder transaction with a CreateCarSellOrderRequest as we did for the car creation transaction request.